doc/installing.rst

```diff
@@ -29,7 +29,7 @@ Instructions
    after installation.
 
 You may optionally use *staging* boards manager package link:
-``http://arduino.esp8266.com/staging/package_esp8266com_index.json``.
+``https://arduino.esp8266.com/staging/package_esp8266com_index.json``.
```

Comment:
Please note that this link doesn't work anymore (no staging packages)

Reply:
Ah, will remove and push.  Wasn't checking that, just the https-ishness of the URLs.